ui/src/components/song/SongCard.tsx

```diff
@@ -44,102 +60,98 @@ const SongCard = (props: SongCardProps) => {
     filterData?.display?.code ?? true,
   ];
   const fieldData = [themes, tempo, originalKey, year, code];
-  const CardFields = ['Themes', 'Tempo', 'Original Key', 'Year', 'Code'];
-
-  // state for the popover, to detect whether mouse is hovering or not
-  const [anchorEl, setAnchorEl] = useState<HTMLElement | null>(null);
+  const navigate = useNavigate();
 
-  const handlePopoverOpen = (event: React.MouseEvent<HTMLElement>) => {
-    setAnchorEl(event.currentTarget);
-  };
-
-  const handlePopoverClose = () => {
-    setAnchorEl(null);
-  };
-
-  const open = Boolean(anchorEl);
+  // state for the modal
+  const [modalOpen, setModalOpen] = useState(false);
+  const handleOpen = () => setModalOpen(true);
+  const handleClose = () => setModalOpen(false);
 
   return (
-    <>
-      <Container
-        sx={{
-          borderRadius: '4px',
-          background: '#FAFAFA',
-          padding: '10px',
-        }}
-      >
-        <Stack direction="row" display="flex" justifyContent="space-between">
-          <Stack>
-            <Typography variant="h2" fontWeight={700} color={'primary.main'}>
-              {title}
-            </Typography>
-            <Typography variant="subtitle1">{artist}</Typography>
-          </Stack>
-          {filterData?.display?.lyricsPreview === true ||
-          filterData?.display?.lyricsPreview === undefined ? (
-            <>
-              <Box
-                aria-owns={open ? 'mouse-over-popover' : undefined}
-                aria-haspopup="true"
-                onMouseEnter={handlePopoverOpen}
-                onMouseLeave={handlePopoverClose}
-                sx={{ height: '30px', width: '30px' }}
-              >
-                <VisibilityIcon sx={{ color: 'primary.main' }} />
-              </Box>
-              <Popover
-                id="mouse-over-popover"
+    <Container
+      sx={{
+        borderRadius: '4px',
+        background: '#FAFAFA',
```

Comment:
should this be in the `theme.js`? can be part of the `secondary` colors? it's grey right